doc/API_REFERENCE.md

```diff
@@ -37,6 +37,7 @@ Install, bootstrap and configure Zend Server.
   $admin_api_key_name   = 'admin',
   $admin_api_key_secret = 'caff756fd7682fa35901afa923822f63771570c25afd5368e',
   $admin_email          = 'admin@domain.tld',
+  $zsurl                = 'http://localhost:10081',
```

Comment:
We moved $zsurl into the module config so it can be defined in the provisioning config.